katharsis-jpa/src/main/java/io/katharsis/jpa/internal/paging/DefaultPagedLinksInformation.java

```diff
@@ -0,0 +1,44 @@
+package io.katharsis.jpa.internal.paging;
+
+public class DefaultPagedLinksInformation implements PagedLinksInformation {
+
+	private String first;
+
+	private String last;
+
+	private String next;
+
+	private String prev;
+
+	public String getFirst() {
+		return first;
+	}
+
+	public void setFirst(String first) {
+		this.first = first;
+	}
+
+	public String getLast() {
+		return last;
+	}
+
+	public void setLast(String last) {
+		this.last = last;
+	}
+
+	public String getNext() {
+		return next;
+	}
+
+	public void setNext(String next) {
+		this.next = next;
+	}
+
+	public String getPrev() {
+		return prev;
+	}
+
+	public void setPrev(String prev) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)